src/pages/Releases/ReleasesPage.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import marked from "marked";
+import { MuiThemeProvider, withStyles } from "@material-ui/core/styles/index";
+import DOMPurify from "dompurify";
+import classNames from "classnames";
+
+import Paper from "@material-ui/core/Paper";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableRow from "@material-ui/core/TableRow";
+import TableCell from "@material-ui/core/TableCell";
+import TableFooter from "@material-ui/core/TableFooter";
+import TablePagination from "@material-ui/core/TablePagination";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+
+import useReleases from "../../hooks/useReleases";
+import Header from "../../components/Header/Header";
+import CONSTANTS from "../../helpers/constants";
+
+const styles = () => ({
+    container: {
+        padding: "6rem"
+    },
+    cardHolder: {
+        border: "none",
+        paddingBottom: "1rem"
+    },
+    isFirstCard: {
```

Comment:
`latestRelease`

Reply:
Will update this so that the style is applied for the latest release card rather than the first card